ci-operator/config/shipwright-io/build/shipwright-io-build-master__4.5.yaml

```diff
@@ -0,0 +1,27 @@
+base_images:
```

Comment:
I think that the two underscores at the end of the filename should be a dash instead.
`ci-operator/config/shipwright-io/build/shipwright-io-build-master-4.5.yaml`

Reply:
Both files under that directory are having the `__` on their names. Should we rename them both later?